src/utils/listUtils.ts

```diff
@@ -0,0 +1,7 @@
+export function uniqBy<T extends any[]>(a: T, keyFn: (_t: T) => string) {
+  var seen: Record<string, boolean> = {};
+  return a.filter((item) => {
+    var k = keyFn(item);
```

Comment:
@aradmargalit Can you remind me why we'd prefer `var` to `let` here? I don't often see `var` in the TS you write

Reply:
I stole this, lemme update it